### PR TITLE
feat: landing page

### PR DESCRIPTION
**Pull request by sethfeingold**
First draft of new landing page

things still needed:

- mobile responsive styling and mobile nav
- smooth scrolling for content section nav

### CODE REVIEW

File: frontend/src/pages/landing-page/index.tsx [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+                    Download with <a href="/">R</a> or <a href="/">Python</a>
+                    <span className={styles.linkArrow}>
+                      <LinkArrow />
+                    </span>
+                  </span>
+                </div>
+              </div>
+            </div>
+            <div className={styles.contentImageCol}>
+              <Image src={DownloadDataImg} alt="" />
```
Comment (tihuan) [inline frontend/src/pages/landing-page/index.tsx:359]: a11y: Can we add description of the image please 🙏 ? Thank you!

here and throughout

File: frontend/src/pages/landing-page/index.tsx [hunk 2]
```diff
@@ -0,0 +1,542 @@
...
+                  data-test-id="cellxgene-logo"
+                  src={Logo}
+                  alt="cellxgene logo"
+                />
+              </a>
+            </Link>
+          </div>
+
+          <div className={styles.footerTopLinks}>
+            <a href="https://github.com/chanzuckerberg/aspen/" target="_blank">
```
Comment (tihuan) [inline frontend/src/pages/landing-page/index.tsx:490]: let's add `rel="noopener"` for security!

here and throughout

source: https://web.dev/external-anchors-use-rel-noopener/

File: frontend/src/pages/landing-page/index.tsx [hunk 3]
```diff
@@ -0,0 +1,542 @@
...
+              Github
+            </a>
+            <a
+              href="https://chanzuckerberg.com/careers/career-opportunities/?initiative=science"
+              target="_blank"
+            >
+              Careers
+            </a>
+            <a href="https://czgenepi.org/resources">Resources</a>
```
Comment (tihuan) [inline frontend/src/pages/landing-page/index.tsx:499]: I know the PR is still WIP, but just wanted to add a reminder that the URLs are all `czgenepi` right now!

**Review by tihuan — COMMENTED** [review]
Thanks so much for the awesome and modern looking landing page! Looks very exciting 🎉 

I know the PR is still WIP, so just a few comments for thoughts 🙏 

Thank you!

**Review by tihuan — COMMENTED** [review]
Also for all the images, can we run them through [sqooush](https://squoosh.app/) for smaller file size please?

**Comment by ccampb13** [comment]
hey team! @sethfeingold is out of the office through tomorrow. Some of the CZI stakeholders are curious if there's a staging link available for preview?

**Comment by ccampb13** [comment]
@seve @tihuan are you able to help there?

**Comment by seve** [comment]
@ccampb13 I can push it to a remote deployment. I'll ping in slack when it's available.

File: frontend/src/pages/landing-page/index.tsx [hunk 4]
```diff
@@ -0,0 +1,542 @@
...
+            <div>
+              <span>donors</span>
+              <p>2.7k+</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <main className={styles.main}>
+        <div className={styles.contentNav}>
+          <Link passHref href="#single-cell">
```
[the remark this answers is not in the document]
Comment (sethfeingold) [reply frontend/src/pages/landing-page/index.tsx:123]: Working on smooth scroll/scroll padding functionality :) I'll be making sure these scroll to the section with enough top spacing to highlight the appropriate section

File: frontend/src/pages/landing-page/index.tsx [hunk 5]
```diff
@@ -0,0 +1,542 @@
...
+  });
+
+  const [activeNav, setActiveNav] = useState(1);
+
+  return (
+    <>
+      <Head>
+        <title>cellxgene | Home</title>
+      </Head>
+      <header className={styles.header}>
```
[the remark this answers is not in the document]
Comment (sethfeingold) [reply frontend/src/pages/landing-page/index.tsx:50]: Tweet logo is fixed! New tweets are going to be in place with correct avatars.

I believe landing page will use the newer innermost header and footer, and the rest of the site will likely be using the original outermost header/footer

File: frontend/src/pages/landing-page/index.tsx (hunk 2 above)
Comment (sethfeingold) [reply frontend/src/pages/landing-page/index.tsx:490]: done!

File: frontend/src/pages/landing-page/index.tsx (hunk 3 above)
Comment (sethfeingold) [reply frontend/src/pages/landing-page/index.tsx:499]: Copy doc still being updated, but looks like these will use cellxgene url's which are going to be completed before launch. example: cellxgene.cziscience.com/privacy

**Pull request merged**